packages/edit-site/src/components/page/header.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	FlexBlock,
+	FlexItem,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+export default function Header( { title, subTitle, actions } ) {
+	return (
+		<HStack as="header" alignment="left" className="edit-site-page-header">
+			<FlexBlock className="edit-site-page-header__page-title">
+				<Heading
+					as="h1"
+					level={ 4 }
```

Comment:
Any reason why we are using level 4? It seems like the heading here might be the most important one so maybe it makes sense to be level 1? 🤔

Reply:
Setting level is just for styling. It's still an H1 for markup reasons.

Reply:
Hmm, yeah, you're right! I think the API is confusing though, I'd imagine that setting `level` changes the `aria-level` attribute but it doesn't. It's a totally different issue though so feel free to ignore me 😆 .